tiup/tiup-component-cluster-edit-config.md

```diff
@@ -0,0 +1,33 @@
+---
+title: tiup cluster edit-config
+---
+
+# tiup cluster edit-config
+
+If you need to modify the cluster configuration after the cluster is deployed, you can use the `tiup cluster edit-config` command that starts a editor for you to modify the topology file<!-- (/tiup/tiup-cluster-topology-reference.md) --> of a cluster. This editor is specified in the `$EDITOR` environment variable by default. If the `$EDITOR` environment variable does not exist, the `vi` editor is used.
```

Comment:
```suggestion
If you need to modify the cluster configuration after the cluster is deployed, you can use the `tiup cluster edit-config` command that starts an editor to modify the topology file<!-- (/tiup/tiup-cluster-topology-reference.md) --> of a cluster. This editor is specified in the `$EDITOR` environment variable by default. If the `$EDITOR` environment variable does not exist, the `vi` editor is used.
```

Reply:
It is "you" who is enabled by the command to modify the file, not the command itself to modify the file.

Reply:
```suggestion
If you need to modify the cluster configuration after the cluster is deployed, you can use the `tiup cluster edit-config` command that starts an editor for you to modify the topology file<!-- (/tiup/tiup-cluster-topology-reference.md) --> of a cluster. This editor is specified in the `$EDITOR` environment variable by default. If the `$EDITOR` environment variable does not exist, the `vi` editor is used.
```

Reply:
@TomShawn rest LGTM